SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -76,8 +76,13 @@ public IActionResult NewOrReturningUser_Post(NewOrReturningUserViewModel viewMod
                     ModelState.AddModelError(nameof(NewOrReturningUserViewModel.Reference), "Check you have typed the reference correctly. Reference must be 8 characters.");
                     return NewOrReturningUser_Get();
                 }
+
+                var userDataModel = userDataStore.LoadUserData(viewModel.Reference);
                 
-                return RedirectToAction("YourSavedRecommendations_Get", "EnergyEfficiency", new { reference = viewModel.Reference });
+                // TODO: Seabeta-346 to combine this with magic link logic
+                return userDataModel.UserRecommendations.Any()
+                    ? RedirectToAction("Recommendation_Get", "EnergyEfficiency", new { reference = viewModel.Reference, id = (int) userDataModel.UserRecommendations[0].Key })
+                    : RedirectToAction("NoRecommendations_Get", "EnergyEfficiency", new { reference = viewModel.Reference });

```

Comment:
I'll handle this and the magic link in the next ticket (no point in doing it now since I'll have to think about it)